practice-areas/accessibility-how-to.md

```diff
@@ -0,0 +1,37 @@
+
+1/ [Accessibility Insights](https://accessibilityinsights.io/) (FastPass)
+
+Microsoft's open source Accessibility Insights offers both the automated testing with axe and guided keyboard only testing within their FastPass. This is roughly equivalent to using WebAim's proprietary [Wave](https://wave.webaim.org/) toolbar with \[Keyboard-only testing\](https://webaim.org/techniques/keyboard/). The main advantage of Accessibility Insights is that it provides assets which make it easier to include in JIRA to replicate the results.
+
+How to install: [Install Accessibility Insights for the Web](https://accessibilityinsights.io/downloads/) in Edge/Chrome browser.
+
+How to use: [Watch this video about basic use](https://www.youtube.com/watch?v=XVvBJoEe4Is&ab_channel=AccessibilityInsights) of this extension. After you've installed this browser extension and enabled it in your dashboard, simply click on the icon and then FastPass.
+
+2/ [Accessibility Insights](https://accessibilityinsights.io/) (Assessment)
+
+By doing an automated scan and keyboard only testing you will catch most but not all accessibility errors. Accessibility Insights Assessments guides you through other steps that should be evaluated when testing new patters. This will catch issues which may have been skipped if only automated tests have been used historically.
+
+How to test: [Watch this video guidance on using the assessment section](https://www.youtube.com/watch?v=XVvBJoEe4Is&ab_channel=AccessibilityInsights) of this extension.
+
+3/ [Screen readers](https://www.sarasoueidan.com/blog/testing-environment-setup/)
+
+Most operating systems have a screen reader built in. One of the most popular for blind users is JAWS, but the expense and customizability presents limitations for users. Most casual accessibility testers will leverage the built-in VoiceOver screen reader on a Mac and the open source [NVDA on Windows](https://www.nvaccess.org/download/) for most testing. If using NVDA, consider contributing from your ProDev to support this great open source project. According to a 2021 by [WebAim survey of screen reader users](https://webaim.org/projects/screenreadersurvey9/), ChromeVox & Windows Narrator combined make up less than 1% of use. It is also worth noting that testing on mobile devices is important and that both iOS (VoiceOver) and Android (TalkBack) have built-in screen readers.
+
+It is very difficult for sighted users to understand how a blind person uses a screen reader. You have to essentially forget the visual layout and learn how to navigate the page content with headers, links and aria-regions. It is also important to observe and learn to empathize with blind users, as they are navigating through the page.
```

Comment:
This is admittedly a nit-picky comment :) But I like to reframe empathy as needing to learn from screen reader users' expertise. I'd change this to: "It is also important to observe and learn from the ways blind users navigate through the page. This video demonstrates [How A Screen Reader User Surfs The Web](https://www.youtube.com/watch?v=OUDV1gqs9GA)."

Reply:
That makes sense to me. Like that you include a link to a Smashing video. So just replace that sentence with line 25 or just part of it?

Reply:
@maggiewachs can you follow up on Mike's suggestion so we can merge this PR?

Reply:
Ok, I've modified it. Now reads as:  
  
`It is very difficult for sighted users to understand how a blind person uses a screen reader. You have to essentially forget the visual layout and learn how to navigate the page content with headers, links and aria-regions. It is also important to observe and learn from the ways blind users navigate through the page. This video demonstrates [How A Screen Reader User Surfs The Web](https://www.youtube.com/watch?v=OUDV1gqs9GA).`